src/functions/slack_function_handler_type_test.ts

```diff
@@ -190,3 +190,51 @@ Deno.test("SlackFunctionHandler with only outputs", () => {
   const result = handler(createContext({ inputs: {} }));
   assertEquals(result.outputs?.out, "test");
 });
+
+Deno.test("SlackFunctionHandler with only completed false", () => {
+  const TestFn = DefineFunction({
+    callback_id: "test",
+    title: "test fn",
+    source_file: "test.ts",
+    output_parameters: {
+      properties: {
+        completed: {
+          type: "boolean",
+        },
+      },
```

Comment:
nit: Naming the output param `completed` is a bit confusing. I think it's a good idea for this test to have output parameters, and actually would be even better if those outputs are required. Let's just name it something else and include it in the `required` array.

The reason for this is that a Successful response would look like
```ts
return {
  outputs: {
    completed: false,
  }
}
```

But what we want to test for is what you have already
```ts
return {
  completed: false,
}
```

IMO these are too similar and might trip someone up.